netket/graph/space_group.py

```diff
@@ -158,7 +166,7 @@ def _full_translation_group(self) -> PermutationGroup:
         )
 
     def translation_group(
-        self, axes: Optional[Union[int, Sequence[int]]] = None
+        self, axes: Optional[Union[int, Iterable[int]]] = None
```

Comment:
of course one should only ever pass a tuple for `axes` but does it hurt to support generic iterables (e.g. numpy/jax arrays)?

Reply:
`Iterable` doesn't need to provide acces by index (`axes[i]`), which I'm pretty sure we'd expect the `axes` argument to support. So I suggest we keep `Sequence`.
```suggestion
        self, axes: Optional[Union[int, Sequence[int]]] = None
```